js/ui/appDisplay.js

```diff
@@ -1665,7 +1665,9 @@ const AppIcon = new Lang.Class({
     },
 
     getDragActor: function() {
-        return this.app.create_icon_texture(Main.overview.dashIconSize);
+        // This is a temporary change not to depend on the dash while
+        // we don't implement our own Drag'n'Drop mechanism in EOS.
+        return this.app.create_icon_texture(64);
```

Comment:
Ah, I see now... this might have been overriden by the force push Roddy did, even after I checked the last commits (but probably failed to compare the whole thing). Fortunately, that was the only change that got lost together with this other one I did following Roddy's review:
```
--- a/js/ui/overviewControls.js
+++ b/js/ui/overviewControls.js
@@ -452,7 +452,7 @@ const ControlsManager = new Lang.Class({
 
         geometry.width -= thumbnailsWidth;
 
-        if (this.actor.get_text_direction() != Clutter.TextDirection.LTR)
+        if (this.actor.get_text_direction() == Clutter.TextDirection.RTL)
             geometry.x += thumbnailsWidth;
 
         this.viewSelector.setWorkspacesFullGeometry(geometry);
```
I'll put them back